### PR TITLE
Add more examples for document

### PR DESCRIPTION
**Pull request by hyrodium**
This PR includes the following updates:
* Fix misleading words
* Update the title of the document
* Fix `@example` blocks in the document
* Fix docstring for `drawifinbounds!`

### CODE REVIEW

**Comment by hyrodium** [comment]
On current `master` branch, we have a jaggy image and invalid `# hide` script.

image

With this PR, these problems are solved.
image

**Review by johnnychen94 — APPROVED** [review]
thank you for these improvements!

File: src/core.jl [hunk 1]
```diff
@@ -340,6 +340,7 @@ Draws a single point after checkbounds() for coordinate in the image.
 Color Defaults to oneunit(T)
 
 """
+drawifinbounds!
```
Comment (johnnychen94) [inline src/core.jl:343]: a nice catch 👍

**Comment by hyrodium** [comment]
On current master branch, the [docs](https://juliaimages.org/ImageDraw.jl/dev/) generation is failing.

image

https://github.com/JuliaImages/ImageDraw.jl/runs/2701413156

I've added a dependency on `Images.jl` in #64, so this problem will be fixed when the PR is merged.

**Pull request merged**